src/main/resources/templates/templateDslBuilder.ftlh

```diff
@@ -329,5 +405,121 @@ public class ${schema.className}Builder {
     }
     </#if>
 
-}
+    public static long[] getLongArray(String[] numbers) {
```

Comment:
These code will be insert everytime in the files whe is not necessary.

Besides is public and I believe is not necessayry